include/language.php

```diff
@@ -18,7 +18,7 @@
  * @phpstan-type LanguageData array{
  *     term_props: array{
  *         language: LanguagePropData,
- *         term_language: LanguagePropData
+ *         term_language?: LanguagePropData
  *     },
```

Comment:
I don't think we need to keep `term_language`, and maybe we could add `&array<...>` (not sure if relevant or helpful though).
```
 *     term_props: array{
 *         language: LanguagePropData
 *     }&array<non-empty-string, LanguagePropData>,
```

Reply:
It's not required indeed, but I chose to leave it to document `term_language` should be here ideally.
But if you prefer, I'll remove it.

Reply:
Muokay.
What do you think about adding `&array<...>`?

Reply:
As we discussed, should be simplified as you said. Done in 4e5f860b.